pkgs/tools/graphics/mangohud/default.nix

```diff
@@ -61,7 +77,7 @@ stdenv.mkDerivation rec {
         glxinfo
         gnugrep
         gnused
-        pciutils
```

Comment:
pciutils is no longer needed: https://github.com/flightlessmango/MangoHud/commit/f8fa73daf2748c69d1cb79916edb9f515e016b99#diff-77743d25e8e941dd6c2a92d819b80cfc171108ef8f088cebdaad2b45ad39c928L612-L613